pyt/vulnerabilities.py

```diff
@@ -170,7 +171,8 @@ def append_node_if_reassigned(
 
 def find_triggers(
     nodes,
-    trigger_words
+    trigger_words,
+    nosec_lines = set()
```

Comment:
Can you please do `nosec_lines=set()` instead of `nosec_lines = set()`? I thought `find_vulnerabilities` was the only function that needed this, but you're right since we do call `find_triggers` once from a test here https://github.com/python-security/pyt/blob/a3b99515c8d90f39c4c943c54db0c6f16449dca3/tests/vulnerabilities_test.py#L96 maybe since we just call it once from tests we can pass in the set in that test.

Reply:
You can see some of these in the https://codeclimate.com/github/python-security/pyt/pull/121 output.